eclipse-collections/src/main/java/org/eclipse/collections/impl/collector/Collectors2.java

```diff
@@ -87,7 +90,14 @@
 
 /**
  * A set of Collectors for Eclipse Collections types and algorithms.
- * Includes converters to{Immutable}{Sorted}{List/Set/Bag/Map/BiMap/Multimap}.
+ *
+ * Includes converter Collectors to{Immutable}{Sorted}{List/Set/Bag/Map/BiMap/Multimap}.
+ * Includes Collectors for select, reject, partition.
+ * Includes Collectors for collect, collect{Boolean/Byte/Char/Short/Int/Float/Long/Double}.
+ * Includes Collectors for makeString, zip, chunk.
+ * Includes Collectors for sumBy{Int/Float/Long/Double}.
+ *
+ * Use these Collectors with @{@link RichIterable#reduceInPlace(Collector)} and @{@link Stream#collect(Collector)}.
```

Comment:
These lines are shown as a single line in Javadoc. Maybe better to use <p> </p> or <br> tags to format them.